### ISSUE TITLE
Feature request: Hacs compatibility?

### ISSUE THREAD

**Issue by isabellaalstrom** [r-renato/hass-xiaomi-mi-flora-and-flower-care#12]
Could you please make this repo compliant to HACS standard? This would benefit everyone, since more people can find your integration and card, and also it get's much easier to stay up to date with your code. Great integration!

Basically the only thing you need to do is follow this folder structure and perhaps put the card in a separate repo: https://hacs.xyz/docs/publish/integration

**Comment by r-renato** [comment]
Hi,
the integration component and the card are one, they can't be divided. The card is designed for this component, so there is no sense in publishing it as a plugin.

**Comment by isabellaalstrom** [comment]
There are plenty of examples of the same type of specific cards that's in there for ease of updating for example. Right now I'm guessing you need to update the whole component.

**Comment by r-renato** [comment]
Yeah, you're right. This ensures that any changes to the component will not cause problems to the card (even if they are temporary).
